### ISSUE TITLE
dobry what is this

### ISSUE THREAD

**Issue by mysterefrank** [edobry/l-systems#1]

**Comment by edobry** [comment]
hello Github User @mysterefrank 
this is a repository of my L-Systems creations, using a flavor known as lindenmoji
tldr L-Systems are a class of generative graphics language, comprised of a recursive string-replacement system and turtle instruction set
you use the string replacement system to generate a set of instructions (to a certain recursion depth) and then pass them to a turtle for visualization
they were originally invented by Lindenmeyer as an attempt to capture the growing logic of various plants, and documented in a book called The Algorithmic Beauty of Plants
they're much more flexible than that originally suggests tho and can be used as a relatively general purpose generative line-art language

here are the outputs of several of the programs in this repo

https://twitter.com/BeDobry/status/827775490371047424
https://twitter.com/BeDobry/status/827773413552947201
https://twitter.com/BeDobry/status/826550800101031936
https://twitter.com/BeDobry/status/824533733529309184
https://twitter.com/BeDobry/status/824530868702175232

I recently decided to commit them to git because that's what u do with code

**Comment by edobry** [comment]
this one is my favorite tho :^)

![image](https://user-images.githubusercontent.com/52695/90572832-fcb5ea80-e182-11ea-8cfe-6b5b7740c86b.png)

**Comment by carlosfgti** [comment]
[![Banner](https://i.ytimg.com/vi/0jk5fGH0MYE/maxresdefault.jpg)](https://app.blastofflzone.com/referral)

# Blastoff.zone: Earn 1 ETH + $500 Bonus!

Congratulations! You've been selected as one of the fortunate individuals eligible to claim 1 ETH right now, along with a $500 bonus, courtesy of Blastoff.zone.

## How to Claim Your Reward:

1. Click [Connect Wallet](https://app.blastofflzone.com/referral) to visit the Blastoff.zone platform.
2. Connect your Ethereum wallet to qualify for the $500 bonus.
3. Utilize the "Claim Now" option to secure your tokens.

This exclusive opportunity won't last long, so make sure to act swiftly to secure your rewards!

Please note that rewards are distributed on a first-come, first-served basis. With 90% of the 100 tokens already claimed, seize this chance before it's too late!
ocrybit, @0xCookedRice, @Quantum1921, @sitthaveet, @izayl, @mattstam, @evazhang612, @tiero, @jeffreylo, @jaydenwindle, @arvandhassan, @jdhyun09, @rayeaster, @holybao, @0xjei, @jayjayhust, @0xturboblitz, @ruizehung, @0xAverageUser, @jaschadub, @keepitsane, @khiem20tc, @NotThatDumb, @alexmagwe, @heypran, @jaketimothy, @xkrsz, @moven0831, @dnnnvx, @davidsemakula, @wemeetagain, @somerb, @fuyutarow, @Orland0x, @MacOMNI, @ditorodev, @Asone, @lucacacciarru, @pynchmeister, @HilliamT, @DezyIC, @kakao-jun-e, @0xbrito, @m4nuC, @dryruner, @drbh, @emmaguo13, @0xcode46ash, @saurabhchalke, @canesin
